File: config/src/main/java/org/wcl/mfn/config/ui/UIConfigResourceResolver.java
package org.wcl.mfn.config.ui;

import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.*;

@Configuration
public  class UIConfigResourceResolver implements WebMvcConfigurer {
    public void addResourceHandlers(final ResourceHandlerRegistry registry) {
        registry.addResourceHandler("/css/**")
                .addResourceLocations("classpath:/static/css/");
    }
}

File: config/src/main/kotlin/org/wcl/mfn/config/ui/UIConfigResourceResolver.kt
package org.wcl.mfn.config.ui

import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.*

@Configuration
open class UIConfigResourceResolver : WebMvcConfigurer {
    override fun addResourceHandlers(registry: ResourceHandlerRegistry) {
        registry.addResourceHandler("/css/**")
            .addResourceLocations("classpath:/static/css/")
    }
}
